### PR TITLE
storage: set write detail for all txn write commands

### LINKED ISSUE #14639: The processing time of the scheduler worker is not tracker for all write commands
## Development Task

The processing time of the scheduler worker is not tracked, resource control currently using write bytes approximates the scheduler worker's CPU time penalty which would make scheduling undesirable

### PR DESCRIPTION
**Pull request by Connor1996**
### What is changed and how it works?

Issue Number: Close #14639

What's Changed:

```commit-message
Set write detail for all txn write commands
```

### Check List 

Tests 

- Unit test

### Release note 

```release-note
Set write detail for all txn write commands
```

### CODE REVIEW

File: src/server/service/kv.rs [hunk 1]
```diff
@@ -2062,12 +2062,13 @@ txn_command_future!(future_prewrite, PrewriteRequest, PrewriteResponse, (v, resp
...
         resp.set_min_commit_ts(v.min_commit_ts.into_inner());
         resp.set_one_pc_commit_ts(v.one_pc_commit_ts.into_inner());
-        GLOBAL_TRACKERS.with_tracker(tracker, |tracker| {
-            tracker.write_scan_detail(resp.mut_exec_details_v2().mut_scan_detail_v2());
-            tracker.write_write_detail(resp.mut_exec_details_v2().mut_write_detail());
-        });
+
     }
     resp.set_errors(extract_key_errors(v.map(|v| v.locks)).into());
+    GLOBAL_TRACKERS.with_tracker(tracker, |tracker| {
```
Comment (glorv) [inline src/server/service/kv.rs:2068]: I think we can move this block into txn_command_future macro to avoid copy it too many times.

Comment (Connor1996) [reply src/server/service/kv.rs]: good idea

**Review by glorv — APPROVED** [review]
LGTM

**Review by nolouch — APPROVED** [review]
lgtm

**Review by tabokie — APPROVED** [review]

**Pull request merged**